### PR TITLE
add default dependencies taken from packages.config

### PR DESCRIPTION
**Pull request by reeflog**

### CODE REVIEW

File: src/main/groovy/com/ullink/NuGetSpec.groovy [hunk 1]
```diff
@@ -78,27 +80,43 @@ class NuGetSpec extends Exec {
...
-                    def fwkFolderVersion = project.msbuild.mainProject.properties.TargetFrameworkVersion.toString().replace('v', '').replace('.', '')
-                    defaultFiles <<= { file(src: artifact.toString(), target: 'lib/net' + fwkFolderVersion) }
+                    def fwkFolderVersion = mainProject.properties.TargetFrameworkVersion.toString().replace('v', '').replace('.', '')
+                    defaultFiles.add({ file(src: artifact.toString(), target: 'lib/net' + fwkFolderVersion) })
             }
-
             appendAndCreateParentIfNeeded('files', defaultFiles)
+
+            def packageConfigFile = new File(
+                    (File) new File(mainProject.projectFile).parentFile,
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetSpec.groovy]: cast not needed AFAICT

Comment (reeflog) [inline src/main/groovy/com/ullink/NuGetSpec.groovy]: Removed + fix unit test

File: src/main/groovy/com/ullink/NuGetSpec.groovy [hunk 2]
```diff
@@ -62,13 +63,14 @@ class NuGetSpec extends Exec {
 
     String supplementDefaultValueOnNuspec(String nuspecString) {
         def final msbuildTaskExists = project.tasks.findByName('msbuild') != null
+        def final packageConfigFileName = 'packages.config'
 
-        def root = new XmlSlurper(false, false).parseText(nuspecString)
+        def GPathResult root = new XmlSlurper(false, false).parseText(nuspecString)
```
Comment (timotei) [inline src/main/groovy/com/ullink/NuGetSpec.groovy]: Isn't it weird to have both `def` and the actual type?

Comment (reeflog) [inline src/main/groovy/com/ullink/NuGetSpec.groovy]: Yes, it is. I'll remove the def.

**Pull request merged**